neurolib/models/multimodel/builder/adex.py

```diff
@@ -0,0 +1,944 @@
+"""
+Adaptive exponential integrate-and-fire mean-field approximation.
+
+Main reference:
+    Augustin, M., Ladenbauer, J., Baumann, F., & Obermayer, K. (2017).
+    Low-dimensional spike rate models derived from networks of adaptive
+    integrate-and-fire neurons: comparison and implementation. PLoS Comput Biol,
+    13(6), e1005545.
+
+Additional reference:
+    Cakan C., Obermayer K. (2020). Biophysically grounded mean-field models of
+    neural populations under electrical stimulation. PLoS Comput Biol, 16(4),
+    e1007822.
+"""
+
+import logging
+import os
+from copy import deepcopy
+
+import numba
+import numpy as np
+import symengine as se
+from h5py import File
+from jitcdde import input as system_input
+
+from ..builder.base.constants import EXC, INH, LAMBDA_SPEED
+from ..builder.base.network import Network, SingleCouplingExcitatoryInhibitoryNode
+from ..builder.base.neural_mass import NeuralMass
+
+DEFAULT_CASCADE_FILENAME = "quantities_cascade.h5"
+
+DEFAULT_PARAMS_EXC = {
+    # number of inputs per neuron from EXC/INH
+    "K_exc": 800.0,
+    "K_inh": 200.0,
+    # postsynaptic potential amplitude for global connectome
+    "c_global": 0.4,
+    # number of incoming EXC connections (to EXC population) from each area
+    "K_exc_global": 250.0,
+    # synaptic time constant EXC/INH
+    "tau_syn_exc": 2.0,  # ms
+    "tau_syn_inh": 5.0,  # ms
+    # internal Fokker-Planck noise due to random coupling
+    "sigma_ext": 1.5,  # mV/sqrt(ms)
+    # maximum synaptic current between EXC/INH nodes in mV/ms
+    "J_exc_max": 2.43,
+    "J_inh_max": -3.3,
+    # single neuron parameters
+    "C_m": 200.0,
+    "g_L": 10.0,
+    # external drives
+    "ext_current": 0.0,
+    "ext_rate": 0.0,
+    # adaptation current model parameters
+    # subthreshold adaptation conductance
+    "a": 15.0,  # nS
+    # spike-triggered adaptation increment
+    "b": 40.0,  # pA
+    "E_A": -80.0,
+    "tau_A": 200.0,
+    "lambda": LAMBDA_SPEED,
+}
+
+DEFAULT_PARAMS_INH = {
+    # number of inputs per neuron from EXC/INH
+    "K_exc": 800.0,
+    "K_inh": 200.0,
+    # postsynaptic potential amplitude for global connectome
+    "c_global": 0.4,
+    # number of incoming EXC connections (to EXC population) from each area
+    "K_exc_global": 250.0,
+    # synaptic time constant EXC/INH
+    "tau_syn_exc": 2.0,  # ms
+    "tau_syn_inh": 5.0,  # ms
+    # internal Fokker-Planck noise due to random coupling
+    "sigma_ext": 1.5,  # mV/sqrt(ms)
+    # maximum synaptic current between EXC/INH nodes in mV/ms
+    "J_exc_max": 2.60,
+    "J_inh_max": -1.64,
+    # single neuron parameters
+    "C_m": 200.0,
+    "g_L": 10.0,
+    # external drives
+    "ext_current": 0.0,
+    "ext_rate": 0.0,
+    "lambda": LAMBDA_SPEED,
+}
+# matrices as [to, from], masses as (EXC, INH)
+# EXC is index 0, INH is index 1
+DEFAULT_ADEX_NODE_CONNECTIVITY = np.array([[0.3, 0.5], [0.3, 0.5]])
```

Comment:
Where exactly do these parameters come from? (What are their analoge in `aln`?) Are these defined by the `c`'s? Maybe substitute?

Reply:
yes, these are the `c`s in the original ALN model. Since the whole design of `MultiModel` is such that we have individual masses, then nodes and then network this is the connectivity matrix for AdEx Node (with one excitatory and one inhibitory mass).

As per names - I will open an issue for this and personally, I would deal with it in a separate PR (as well as with other `MultiModel` models)